cmd/validate/image_test.go

```diff
@@ -754,12 +752,10 @@ func Test_ValidateImageCommandExtraData(t *testing.T) {
       custom_rule_data:
         prefix_data:
           - registry1
-configuration:
-  collections:
-    - minimal
-  include:
-    - "*"
-  exclude: []
+    config:
+      include:
+        - '@minimal'
```

Comment:
Not sure what the behavior is when both legacy "collections" and "include" are set, but should this be as follows?

```suggestion
        - '*'
```

Reply:
Maybe it's inconsequential.

Reply:
That's a good question. Arguably, even the old example was a bit odd. I think this probably better captures the intent.